packages/form-js-editor/src/features/properties-panel/entries/AcceptEntry.js

```diff
@@ -0,0 +1,65 @@
+import { get } from 'min-dash';
+
+import { useService, useVariables } from '../hooks';
+
+import { FeelTemplatingEntry, isFeelEntryEdited } from '@bpmn-io/properties-panel';
+
+export function AcceptEntry(props) {
+  const { editField, field } = props;
+
+  const entries = [];
+
+  entries.push({
+    id: 'accept',
+    component: Accept,
+    editField: editField,
+    field: field,
+    isEdited: isFeelEntryEdited,
+    isDefaultVisible: (field) => field.type === 'filepicker',
+  });
+
+  return entries;
+}
+
+function Accept(props) {
+  const { editField, field, id } = props;
+
+  const debounce = useService('debounce');
+
+  const variables = useVariables().map((name) => ({ name }));
+
+  const path = ['accept'];
+
+  const getValue = () => {
+    return get(field, path, '');
+  };
+
+  const setValue = (value) => {
+    return editField(field, path, value);
+  };
+
+  return FeelTemplatingEntry({
+    debounce,
+    element: field,
+    getValue,
+    id,
+    label: 'Supported file formats',
+    singleLine: true,
+    setValue,
+    variables,
+    description,
```

Comment:
A regex validation would be good to have here. Especially if you plan to inject it into the HTML. I'm sure you're going to sanitize it somewhere down the line, but it may become confusing for a user if say they use a ; or something and it doesn't work without a properties panel error.

Reply:
The validation here is less about the risk and more about the UX in my view. If there is any risk to be honest it needs to be handled later anyways because of the feel variables.